data/i18n/hydrogen_fr.ts

```diff
@@ -768,7 +768,7 @@ B</translation>
     </message>
     <message>
         <source>The Timeline is only available in Song Mode</source>
-        <translation>La base de temps est uniquement disponible en mode morceau></translation>
+        <translation>La base de temps est uniquement disponible en mode morceau&gt;</translation>
```

Comment:
the "morceau>" should be "morceau" (my bad probably)
@theGreatWhiteShark : could you please remove the &gt; here? Alternatively, I'll try to remember this and remove it after it'll be merged

Reply:
Sure